types/react-faux-dom/index.d.ts

```diff
@@ -0,0 +1,29 @@
+// Type definitions for react-faux-dom 3.1
+// Project: https://github.com/Olical/react-faux-dom
+// Definitions by: Ali Taheri Moghaddar <https://github.com/alitaheri>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+import * as React from "react";
+
+export class Element extends HTMLElement {
+    style: any;
+
+    constructor(nodeName: string, parentNode?: Element);
+
+    toReact(): React.ReactElement<any>;
+}
+
+export const defaultView: {
+    getComputedStyle(node: Element): { getPropertyValue(name: string): string };
+};
+
+export namespace mixins {
+    const core: any;
```

Comment:
There is no way that such typing can be useful. Because P and S will have to be any. I tried typing these 2 but there are two things to consider:

1. react mixins are on their way out of the ecosystem.
2. typing them is useless. since it's impossible to implicitly add type information to a class created by `createClass`. React doesn't even support mixins with es2015 classes.

I only put these here for completeness.